src/completion/CamelApplicationLauncherTasksCompletionItemProvider.ts

```diff
@@ -108,6 +108,46 @@ export class CamelApplicationLauncherTasksCompletionItemProvider implements vsco
 	"isBackground": true // Must be set as background as the Maven commands doesn't return until the Camel application stops. 
 }`
 	};
+
+	private mavenQuarkusBuildNativeCompletion: vscode.CompletionItem = {
+		label: 'Build a Camel Quarkus application as a Native executable',
```

Comment:
I think we need to add something related to debug:
`Build a Camel Quarkus application as a Camel debuggable Native executable`
`Build a Camel Quarkus application as a Native executable debug-ready`